cmd/kogito/command/project/new_project.go

```diff
@@ -95,9 +96,9 @@ func (i *newProjectCommand) Exec(cmd *cobra.Command, args []string) error {
 
 		log.Infof("Project '%s' created successfully", ns.Name)
 
-		install := shared.ServicesInstallationBuilder(i.Client, ns.Name).SilentlyInstallOperator()
+		install := shared.ServicesInstallationBuilder(i.Client, ns.Name).SilentlyInstallOperatorIfNotExists()
```

Comment:
my point is that we have a different behavior here but the same problem would appear then if Infinispan/Strimzi CRDs are not there or ?

Reply:
We have different behaviors in code for

- `new_project --install-data-index`
- `use_project --install-data-index`
- `install data-index`

shouldn't we unify that ?

Reply:
No, because `new-project` and `use-project` installs a default Data Index. And in #318 I have unified all the work. I need this one to get merged to continue the refactor there.

> my point is that we have a different behavior here but the same problem would appear then if Infinispan/Strimzi CRDs are not there or ?

No, won't appear because the Data Index will get deployed and the operator controller will handle the Infinispan/Kafka installation.

Reply:
The idea here is just to inform the user that the dependencies might not be there and what to expect.

That's one of the reasons why I advocate to remove this installation/dependecy from our operator.

Reply:
oki